app/component/MainMenu.js

```diff
@@ -77,23 +81,55 @@ function MainMenu(props, { config, intl }) {
             </Link>
           </div>
         )}
-        {config.appBarLink && config.appBarLink.name && config.appBarLink.href && (
-          <div className="offcanvas-section">
-            <a
-              id="appBarLink"
-              href={config.appBarLink.href}
-              onClick={() => {
-                addAnalyticsEvent({
-                  category: 'Navigation',
-                  action: 'appBarLink',
-                  name: null,
-                });
-              }}
-            >
-              {config.appBarLink.name}
-            </a>
-          </div>
-        )}
+        {config.mainMenu.countrySelection &&
+          config.mainMenu.countrySelection.map(country => (
+            <div key={country} className="offcanvas-section">
+              <FormattedMessage
+                id={`include-${country}`}
+                defaultMessage={`include-${country}`}
```

Comment:
Maybe default message could be translated include + country code. Would work automatically for all.